include/riak_kv_ts.hrl

```diff
@@ -59,7 +59,14 @@
 
 -record(riak_sql_describe_v1,
         {
-          'DESCRIBE' = <<>>  :: binary()
+          'DESCRIBE'    = <<>>  :: binary()
+        }).
+
+-record(riak_sql_insert_v1,
+        {
+          'INSERT'      = <<>>  :: binary(),
+          fields                :: [{identifier, binary()}],
+          values                :: list(list(term()))
```

Comment:
Really? Don't we break out `integer()` and `float()`, for example?  We also support lists of lists of `list()`?
```erlang
-type ldbvalue() :: binary() | number() | boolean() | list().
```